GitHub: Improve `README.md` (#8)
WindowsAppCommunity/WindowsAppCommunity.Sdk: README.md

> [!WARNING]
> This is a **work in progress** and is not yet ready for use. Please check back later.

<p align="center">
  <img alt="Hero Image" src="https://github.com/user-attachments/assets/ac6d3215-1633-4f09-af31-4203176e904d" />
</p>

<p align="center">
  <a style="text-decoration:none" href="https://windowsappcommunity.com/">

>     <img src="https://img.shields.io/badge/Community-Website-teal" alt="WinAppCommunity Website" /></a>

Reviewer: ```suggestion
    <img src="https://img.shields.io/badge/Community-Website-teal" alt="WindowsAppCommunity Website" /></a>
```

>   <a style="text-decoration:none" href="https://www.nuget.org/packages/WinAppCommunity.Sdk">

Reviewer: ```suggestion
  <a style="text-decoration:none" href="https://www.nuget.org/packages/WindowsAppCommunity.Sdk">
```

>     <img src="https://img.shields.io/nuget/v/WinAppCommunity.Sdk.svg" alt="Version" /></a>

Reviewer: ```suggestion
    <img src="https://img.shields.io/nuget/v/WindowsAppCommunity.Sdk.svg" alt="Version" /></a>
```

  <a style="text-decoration:none" href="https://dsc.gg/uwpc">
    <img src="https://img.shields.io/discord/372137812037730304?label=Discord&color=7289da" alt="WinAppCommunity Discord" /></a>
</p>

Introducing the **Windows App Community SDK**, a library that enables you to manage your membership, projects and publisher data in the Windows App Community.

> The `WinAppCommunity.Sdk` library was created to supersede the original [`uwpcommunity-backend`](https://github.com/WindowsAppCommunity/uwpcommunity-backend) (known to many as **Server Companion**) that has served the community for over 5 years.

Reviewer: ```suggestion
The `WindowsAppCommunity.Sdk` library was created to supersede the original [`uwpcommunity-backend`](https://github.com/WindowsAppCommunity/uwpcommunity-backend) (known to many as **Server Companion**) that has served the community for over 5 years.
```

> The "new infra" as this library is often referred to by many members of the community, has been in the works for ages and we're pleased to announce that it's in rapid development.

Reviewer: ```suggestion
The "new infra", as this library is often referred to by many members of the community, has been in the works for ages. We're pleased to announce that it's now in rapid development.
```

> Built using the **Inter-Planetary File System**, the new community infra boasts powerful features such as <!-- sorry got cut off while writing this -->

Reviewer: ```suggestion
Built using the content-addressing capabilities of [`ipfs`](https://ipfs.tech/), a powerful p2p protocol alternative to `http`, the new community infra boasts powerful features such as... [todo, come back soon]
```

<!--
## Features

- **x**: 
- **y**: 
- **z**:
-->

## Usage

### Installing the SDK

> Published releases are available on [NuGet](https://www.nuget.org/packages/WinAppCommunity.Sdk). To install, run the following command in the [Package Manager Console](https://docs.nuget.org/docs/start-here/using-the-package-manager-console):

Reviewer: ```suggestion
Published releases are available on [NuGet](https://www.nuget.org/packages/WindowsAppCommunity.Sdk). To install, run the following command in the [Package Manager Console](https://docs.nuget.org/docs/start-here/using-the-package-manager-console):
```

```ps1

> PM> Install-Package WinAppCommunity.Sdk

Reviewer: ```suggestion
PM> Install-Package WindowsAppCommunity.Sdk
```

``` 
Or using the [dotnet](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet) CLI

```ps1

> > dotnet add package WinAppCommunity.Sdk

Reviewer: ```suggestion
> dotnet add package WindowsAppCommunity.Sdk
```
